examples/MvcWeb/TheraLang/Controllers/ResourceController.cs

```diff
@@ -0,0 +1,53 @@
+﻿using MvcWeb.TheraLang.Entities;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using MvcWeb.TheraLang.Services;
+
+namespace MvcWeb.TheraLang.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ResourceController : ControllerBase
+    {
+        public ResourceController(IResourceService _service)
+        {
+            this.service = _service;
+        }
+
+        private readonly IResourceService service;
+
+        [HttpPost]
+        public async Task<IActionResult> PostResource(Resource resource)
+        {
+            if(!ModelState.IsValid)
+            {
+                return BadRequest();
+            }
+
+            bool result = await service.AddResource(resource);
+            if(!result)
+            {
+                return BadRequest();
+            }
```

Comment:
do not do that - this is a **bad practise** we should throw an error in the service if something went wrong
in your case - we wouldn't have even exception logs if something happen:
![image](https://user-images.githubusercontent.com/8735059/68190075-6b8bd380-ffb5-11e9-805a-4b025379ed01.png)
you should do something like this:
![image](https://user-images.githubusercontent.com/8735059/68190992-2e284580-ffb7-11e9-95f2-057ca8d30121.png)
note: here resource have toString ovverrided